kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.testing.shape;
+
+import java.util.ArrayList;
+
+public class ShapeCollector {
+
+    public ArrayList<Shape> shapeList = new ArrayList<Shape>();
+    Shape shape;
+
+    public void addFigure(Shape shape){
+        shapeList.add(shape);
+
+    }
+
+    public boolean removeFigure(Shape shape){
+        boolean result = false;
+
+        if (shapeList.contains(shape)){
+            shapeList.remove(shape);
+            result = true;
+        }
+        return result;
```

Comment:
shapeList.remove(shape) zwraca booleana i robi to samo co twoja metoda, tzn,
```
    public boolean removeFigure(Shape shape) {
        return shapeList.remove(shape);
    }
```
bedzie robila dokladnie to samo